src/test/java/uk/gov/hmcts/reform/sscs/json/RoboticsJsonValidatorTest.java

```diff
@@ -1,81 +1,157 @@
 package uk.gov.hmcts.reform.sscs.json;
 
-import com.google.common.collect.ImmutableList;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Map;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.everit.json.schema.ValidationException;
 import org.json.JSONObject;
+import org.json.JSONTokener;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 
+@RunWith(JUnitParamsRunner.class)
 public class RoboticsJsonValidatorTest {
 
-    private RoboticsJsonValidator roboticsJsonValidator = new RoboticsJsonValidator("/schema/sscs-robotics.json");
+    JSONObject jsonData = new JSONObject(
+            new JSONTokener(getClass().getResourceAsStream("/schema/valid_robotics_agreed.json")));
+
+    private RoboticsJsonValidator roboticsJsonValidator = new RoboticsJsonValidator(
+            "/schema/sscs-robotics.json");
+
+    @Test
+    public void givenValidInputAgreedWithAutomationTeam_thenValidateAgainstSchema() throws ValidationException {
+        roboticsJsonValidator.validate(jsonData);
+    }
+
+    @Test
+    @Parameters({"Yes", "No"})
+    public void givenJsonWithTheSameAddressAsAppellantProperty_shouldValidateSuccessfully(String value) {
+        jsonData.getJSONObject("appointee").put("sameAddressAsAppellant", value);
+        roboticsJsonValidator.validate(jsonData);
+    }
 
     @Test
-    public void validateValidRoboticsJson() {
```

Comment:
Are these tests no longer needed?

Reply:
Yes, if you remember our conversation. We said to merged those 2 test classes into one which I did and then I remove the duplicate tests.